content/python/concepts/built-in-functions/terms/divmod/divmod.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'divmod()'
+Description: 'Returns the quotient and remainder of the division of two numbers.'
+Subjects:
+
+    'Computer Science'
+    'Mathematics'
+    Tags:
+    'Functions'
+    'Math'
+    'Numbers'
+    CatalogContent:
+    'learn-python-3'
+    'paths/computer-science'
+    'paths/mathematics'
```

Comment:
please reference a template or an existing entry for the formatting in the metadata section. There should not be any empty lines and items listed should use the list syntax. Also, catalog content should only include two items. Please delete one.

Reply:
In the case of catalog content only containing 2 items, I'll update the other ones in the terms folder with 3 items as well if you'd like? e.g.: dict, delattr, complex, ...

Reply:
I however already changed the divmod.md to your liking :wink: